public/src/components/channelManagement/bannerDesigns/BannerDesignForm.tsx

```diff
@@ -108,101 +134,122 @@ const BannerDesignForm: React.FC<Props> = ({
 
   return (
     <div className={classes.container}>
-      <div className={classes.sectionContainer}>
-        <Typography variant={'h3'} className={classes.sectionHeader}>
+      <Accordion className={classes.accordion}>
```

Comment:
It's a messy diff, but this essentially just swaps each div/Typography for an Accordion

Reply:
👍 It's a good use case for [hiding whitespace](https://github.com/guardian/support-admin-console/pull/508/files?diff=unified&w=1), the diff definitely gets simpler!